### DISCUSSION TITLE
Troubleshooting skipped job

### DISCUSSION THREAD

**Discussion by aeryaguzov** [community/community#25648] [Actions]
<p>Hi,</p>
<p>For some reason I have a job that runs for 20-30min, then terminates as skipped and I can see “skipped 5 minutes ago in 24m 34s”.</p>
<p>I can see logs from this job (unless I refresh the page), so it is not “if” statement issue.</p>
<p>However when I refresh the page logs dissapear and I can see yellow spinner near the step on which it was terminated (which means step is in progress).</p>
<p>Is there any way to troubleshoot this issue ?</p>

**Comment by ChristopherHX** [comment, marked as answer]
<p>I’m pretty shure the problem with your job is that the <a href="https://github.com/actions/runner" class="inline-onebox" rel="noopener nofollow ugc">GitHub - actions/runner: The Runner for GitHub Actions</a> Runner.Worker process died or lost the connection to github.<br>
It might be possible that your job caused the github hosted runner to run out of diskspace, then the runner died.</p>
<p>I’m able to force this ui behavior<br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/4/c/4c69ccefa9c3960d8ebf2ee2875906e7804595a6.png" data-download-href="/uploads/short-url/aTYXMXeJtvijfkrzhTgLsaPjquy.png?dl=1" title="Screenshot_20210823-232524~2" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/44845461/181092340-c4abc9bc-982f-4748-847e-e050c43fe494.png" alt="Screenshot_20210823-232524~2" data-base62-sha1="aTYXMXeJtvijfkrzhTgLsaPjquy" width="345" height="149" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/4/c/4c69ccefa9c3960d8ebf2ee2875906e7804595a6_2_345x149.png, https://aws1.discourse-cdn.com/github/optimized/3X/4/c/4c69ccefa9c3960d8ebf2ee2875906e7804595a6_2_517x223.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/4/c/4c69ccefa9c3960d8ebf2ee2875906e7804595a6_2_690x298.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/4/c/4c69ccefa9c3960d8ebf2ee2875906e7804595a6_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screenshot_20210823-232524~2</span><span class="informations">2560×1112 135 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>The worker uploads the log of the current running step after it finished, in your case it never finished correctly and only live logs were visible. Github discards live logs after streaming then to the UI, so they are gone if you reload the page.</p>
<p>I wonder why such a job is marked as skipped in this situation, it clearly failed.</p>
<blockquote>
<p>Is there any way to troubleshoot this issue ?</p>
</blockquote>
<p>If you used a self-hosted runner upload / look into the files in the _diag folder of the runner root folder. These files might contain more information about your issue.<br>
Rerun your job, if this works then it wasn’t an issue with your workflow, but with the runner of github actions.</p>
